aip/general/0127.md

```diff
@@ -54,8 +54,9 @@ message CreateBookRequest {
 
   // The user-specified ID for the book.
   // When using HTTP/JSON, this field is populated based on a query string
-  // argument, such as `?book_id=foo`. This is the fallback for fields that
+  // argument, such as `?bookId=foo`. This is the fallback for fields that
   // are not included in either the URI or the body.
+  // Note that the field names are converted to camelCase.
```

Comment:
Do we want to write this statement in active voice?

Note that clients convert the filed names to into camelCase format.

Reply:
Done!